themes/theme-ios11/pages/index.jsx

```diff
@@ -38,49 +39,53 @@ import locale from '../locale';
 import themeApi from '../themeApi';
 import * as routes from './routes';
 
+const themeConfigParse = new ConfigParser(themeConfig).parse();
+
 /**
  * The theme's main component defines all the routes (views) inside the application.
  * @returns {JSX}
  */
 const Pages = ({ store }) => (
   <App locale={locale} store={store}>
-    <NavigationHandler>
-      <AppContext.Provider value={{ ...appConfig }}>
-        <ThemeContext.Provider value={themeApi}>
-          <LoadingProvider>
-            <ToastProvider>
-              <Portal name={APP_GLOBALS} />
-              <Viewport>
-                <ModalContainer component={Dialog} />
-                <Toaster render={props => <SnackBar {...props} />} />
-                <Router history={history}>
-                  <Route pattern={INDEX_PATH} component={routes.StartPage} />
-                  <Route pattern={PAGE_PATTERN} component={routes.Page} />
-                  <Route pattern={CATEGORY_PATTERN} component={routes.Category} cache />
-                  <Route pattern={CATEGORY_FILTER_PATTERN} component={routes.Filter} />
-                  <Route pattern={ITEM_PATTERN} component={routes.Product} />
-                  <Route pattern={ITEM_GALLERY_PATTERN} component={routes.ProductGallery} />
-                  <Route pattern={ITEM_REVIEWS_PATTERN} component={routes.Reviews} />
-                  <Route pattern={ITEM_WRITE_REVIEW_PATTERN} component={routes.WriteReview} />
-                  <Route pattern={CART_PATH} component={routes.Cart} />
-                  <Route pattern={BROWSE_PATH} component={routes.Browse} />
-                  <Route pattern={MORE_PATH} component={routes.More} />
-                  {
-                    appConfig.hasFavorites
-                    && <Route pattern={FAVORITES_PATH} component={routes.Favorites} />
-                  }
-                  <Route pattern={LOGIN_PATH} component={routes.Login} />
-                  <Route pattern={SEARCH_PATTERN} component={routes.Search} cache />
-                  <Route pattern={SEARCH_FILTER_PATTERN} component={routes.Filter} />
-                  <Route pattern={SCANNER_PATH} component={routes.Scanner} />
-                  {React.Children.map(routePortals, Component => Component)}
-                </Router>
-              </Viewport>
-            </ToastProvider>
-          </LoadingProvider>
-        </ThemeContext.Provider>
-      </AppContext.Provider>
-    </NavigationHandler>
+    <ConfigContext.Provider value={themeConfigParse}>
```

Comment:
Same as above as well (unnecessary if the theme config would not diverge from `appConfig.theme`).

Reply:
Same answer as above!